model/src/main/kotlin/io/spine/validation/ValidationPlugin.kt

```diff
@@ -51,7 +52,12 @@ import io.spine.validation.required.RequiredPolicy
  * The concrete implementations should provide [renderers], which implement
  * these constraints for a specific programming language.
  */
-public abstract class ValidationPlugin(renderers: List<Renderer<*>> = emptyList()) : Plugin(
+public abstract class ValidationPlugin(
```

Comment:
Having this type as `public` is now confusing. Given we don't suggest that end-users create their own plugins, but rather extend via `CustomOption`, this type should be `@Internal` at the very least.

Reply:
It was `public` before, I haven't changed its visibility.

It is `public` for consumers of the `:model` module, which are language-specific plugins.